tests/iterator-tests.lisp

```diff
@@ -178,14 +178,35 @@
   (is (== (iter:collect! (iter:filter-map! string:parse-int (iter:into-iter (make-list "1" "2" "three" "" "5"))))
           (make-list 1 2 5))))
 
+(define-test iter-for-each ()
+  (let ((populate-vector (fn ()
+			   (let v = (vector:with-capacity 6))
+			   (iter:for-each! (fn (x)
+					     (vector:push! x v)
+					     Unit)
+					   (iter:up-to 5))
+			   v)))
+    (is (== (iter:collect! (iter:into-iter (populate-vector)))
+	    (make-list 0 1 2 3 4)))))
+
+(define-test iter-count ()
+  (is (== (iter:count! (iter:into-iter (make-list 1 2 3 4 5)))
+	  5)))
+
+(define-test iter-find ()
+  (let ((even-in-iter (fn (iter)
+			(iter:find! (fn (x) (coalton-library/math/arith:zero? (mod x 2)))
```

Comment:
This should be written as `math:zero?`

Reply:
All of the `math/*` packages get pulled together [here](https://github.com/coalton-lang/coalton/blob/main/library/prelude.lisp#L5-L16).